Modules/Quotas/src/Quotas.Application/Infrastructure/Persistence/IQuotasDbContext.cs

```diff
@@ -0,0 +1,7 @@
+﻿using Enmeshed.BuildingBlocks.Application.Abstractions.Infrastructure.Persistence.Database;
+
+namespace Backbone.Modules.Quotas.Application.Infrastructure.Persistence;
+
+public interface IQuotasDbContext : IDbContext
```

Comment:
We don't need the IQuotasDbContext in this module, since we will only you the repositories (we will change that in the other modules as well, sooner or later)